app/helpers/view_helper.rb

```diff
@@ -33,6 +33,11 @@ def govuk_back_link_to(url = :back, body = 'Back')
     link_to(body, url, class: classes)
   end
 
+  def govuk_footer_link_to(body = nil, url = nil, html_options = {}, &block)
+    html_options[:class] = prepend_css_class('govuk-footer__link', html_options[:class])
+    govuk_link_to(body, url, html_options, &block)
```

Comment:
I think we need something similar to other methods here as well, to make sure we set the correct class if a block is given:
```
def govuk_footer_link_to(body = nil, url = nil, html_options = {}, &block)
    if block_given?
      html_options = url
      url = body
    end

    html_options[:class] = prepend_css_class('govuk-footer__link', html_options[:class])

    return govuk_link_to(url, html_options) { yield } if block_given?

    govuk_link_to(body, url, html_options)
  end
```